CONTRIBUTING.md

```diff
@@ -59,9 +56,28 @@ If your code requires new dependencies, be sure that those are available on all
 
 ### User experience
 
-Jamulus is used by people all over the world with different backgrounds and knowledge. Features and language should be user focused.
-The Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
-Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+Jamulus is used by people all over the world with different backgrounds and knowledge. Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+
+Language wise, follow the [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) to remain consistent. UI wise, prefer standard approaches to exotic ones.
+
+### Submitting code
+
+- Create a Github fork of the Jamulus code and create a pull request from that fork when you are done. Please don't use diff/patch to send your code changes. GitHub can be confusing, so feel free to ask for help. The [git-scm book](https://git-scm.com/book/en/v2) might be a good reference to get used to git.
```

Comment:
This should mention _branching_ on the fork, too.  PRs from `master` on the fork are... not great.

Reply:
What would you put here exactly?

Reply:
Create a Github fork of the Jamulus code and in that fork create a branch for your work. You can create a pull request from that branch when you are done. Please don't code on the master branch and don't use diff/patch to send your code changes. GitHub can be confusing, so feel free to ask for help.

Reply:
BTW, is this PR starting to overlap with information about using git, branching, etc. in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md)? Might want to check that anything we say there isn't contradicting what we say here. Ideally, we'd only say things in one place...

Reply:
Hmm. Yes. This would mean TRANSLATING.md should link to this file for rebasing

Reply:
Or have a common "Using Github" include?

Reply:
> Or have a common "Using Github" include?

I doubt the docs in the repo support that. Just the website supports includes

Reply:
You always can include the html url, most modern text editors/viewers do support opening html links.

Reply:
Let's expand the TRANSLATING.md file, remove all or most git related stuff from CONTRIBUTING.md and add a link.

Reply:
>  "Using Github" include?

a USING_GITHUB.md ?

Reply:
I'd be a bit worried about an explicit file named like that since it might imply being another GitHub tutorial.